docs/cloud/data-ingestion.md

```diff
@@ -21,3 +21,83 @@ If your Graphite stack does not currently contain any relay, then you can simply
 
 When creating a Hosted Metrics Graphite instance, we provide a carbon-relay-ng config file that you can plug in and be ready to use out of the box.
 We also have Grafana Labs engineers ready to advise further on set up, if needed.
+
+## Using carbon-relay-ng as a replacement for carbon-relay or carbon-cache
+
+The most simple way to send your carbon traffic to GrafanaCloud is to use carbon-relay-ng as a replacement for your current carbon-relay or carbon-cache. Carbon-relay-ng has a carbon input which supports the plain text and the pickle protocols, just like carbon-relay and carbon-cache. 
+
+An example configuration to do that can be downloaded from your Hosted Metrics instance's "Details" page. It contains a `grafanaNet` route pointing at your instance.
+
+## Sending a copy of your carbon traffic to GrafanaCloud
+
+It is possible to duplicate your data to send one copy to your existing Graphite infrastructure and the other to GrafanaCloud. To do this you can put an instance of carbon-relay-ng in front of your existing carbon-relay or carbon-cache and make it duplicate the traffic. Carbon-relay-ng allows you to specify routes of various types, to send a copy to GrafanaCloud you need to add a route of the type `grafanaNet`, to send a copy to your existing carbon-relay/carbon-cache you can add a carbon route.
```

Comment:
I think prospects may be skeptical of doing this, and would feel more safe putting relay-ng *behind* their existing python relay (if they already use the python relay). maybe we should mention both.

Reply:
possible, the thing is just that explaining how to make the carbon-relay duplicate the traffic to crng, instead of making crng duplicate the traffic, is more complicated because the way to do it depends on the clients current setup. I can still do it though